src/Model/Concerns/HasArrayAccess.php

```diff
@@ -2,7 +2,7 @@
 
 declare(strict_types=1);
 
-namespace atk4\data;
+namespace atk4\data\Model\Concerns;
```

Comment:
I do not like `Concerns`. If this is expected to be depracated, then simply `Deprecated`, but I am not sure on this (I personally do not use it and it should be never used across Atk toolkit).